### PR TITLE
Release Wazuh 3.12.0_7.6.1

### LINKED ISSUE #49: Wazuh Cloudformation Release 3.12.0_7.6.1
Wazuh version: 3.12.0
Elastic version: 7.6.1

### Tasks

- [x] Adapt to new versions (3.12.0 - 7.6.1)

- [x] Check templates and update them if needed

- [x] Tests

- [] Tag: v3.12.0_7.6.1

- [] Draft release

### PR DESCRIPTION
**Pull request by jm404**
Hi team,

This PR closes #49 

Best regards,

Jose

### CODE REVIEW

File: elastic/wazuh_cf_kibana.sh [hunk 1]
```diff
@@ -260,7 +260,7 @@ get_plugin_url(){
       BUILD_SRC=$(pwd)
       APP_FILE=$(ls *.zip)
     else
-      plugin_url="https://aws-nist.s3-us-west-1.amazonaws.com/wazuhapp-3.11-sources.zip"
+      plugin_url="https://aws-nist.s3-us-west-1.amazonaws.com/wazuhapp-3.12-sources.zip"
```
Comment (manuasir) [inline elastic/wazuh_cf_kibana.sh]: This resource is down.

File: elastic/wazuh_cf_kibana.sh [hunk 2]
```diff
@@ -271,15 +271,18 @@ install_plugin(){
...
+    cd /usr/share/kibana
     sudo -u kibana /usr/share/kibana/bin/kibana-plugin install file://$BUILD_SRC/$APP_FILE
   fi
+  cd /tmp
   echo "App installed!" >> /tmp/deploy.log
   echo "Redirecting to Wazuh app " >> /tmp/deploy.log
   # Set Wazuh app as the default landing page
-  echo "server.defaultRoute: /app/wazuh" >> /etc/kibana/kibana.yml
-  # Redirect Kibana welcome screen to Discover
+  #echo "server.defaultRoute: /app/wazuh" >> /etc/kibana/kibana.yml
```
Comment (manuasir) [inline elastic/wazuh_cf_kibana.sh]: Please describe the reason why this was removed.

File: splunk/wazuh_cf_splunk.sh [hunk 3]
```diff
@@ -11,8 +11,8 @@ eth0_ip=$(/sbin/ifconfig eth0 | grep 'inet addr:' | cut -d: -f2  | cut -d' ' -f1
 wazuh_api_user=$(cat /tmp/wazuh_cf_settings | grep '^WazuhApiAdminUsername:' | cut -d' ' -f2)
 wazuh_api_password=$(cat /tmp/wazuh_cf_settings | grep '^WazuhApiAdminPassword:' | cut -d' ' -f2)
 wazuh_api_port=$(cat /tmp/wazuh_cf_settings | grep '^WazuhApiPort:' | cut -d' ' -f2)
-TAG="v3.11.4"
-APP_TAG="v3.11.4-7.3.4"
+TAG="v3.12.0"
+APP_TAG="v3.12.0-7.3.4"
```
Comment (manuasir) [inline splunk/wazuh_cf_splunk.sh]: The tag should be `v3.12.0-7.6.1`

File: wazuh/cluster/wazuh_cf_master.sh [hunk 4]
```diff
@@ -26,7 +26,7 @@ AwsSecretKey=$(cat /tmp/wazuh_cf_settings | grep '^AwsSecretKey:' | cut -d' ' -f
 AwsAccessKey=$(cat /tmp/wazuh_cf_settings | grep '^AwsAccessKey:' | cut -d' ' -f2)
 SlackHook=$(cat /tmp/wazuh_cf_settings | grep '^SlackHook:' | cut -d' ' -f2)
 EnvironmentType=$(cat /tmp/wazuh_cf_settings | grep '^EnvironmentType:' | cut -d' ' -f2)
-TAG='v3.11.4'
+TAG='v3.12.4'
```
Comment (manuasir) [inline wazuh/cluster/wazuh_cf_master.sh]: Tag `v3.12.4` doesn't exist.

File: splunk/wazuh_cf_splunk.sh (hunk 3 above)
Comment (jm404) [reply splunk/wazuh_cf_splunk.sh]: Splunk follows different tagging versions: https://github.com/wazuh/wazuh-splunk/tree/v3.12.0-7.3.4

File: elastic/wazuh_cf_kibana.sh (hunk 1 above)
Comment (jm404) [reply elastic/wazuh_cf_kibana.sh]: Fixed in fdde6b02bbceb82bdd4a4ce4be153f83c2162603

File: elastic/wazuh_cf_kibana.sh (hunk 2 above)
Comment (jm404) [reply elastic/wazuh_cf_kibana.sh]: Testing typo, fixed in 9fe55910f91e2df6b406f8a17329fd99cfa3f208

File: wazuh/cluster/wazuh_cf_master.sh (hunk 4 above)
Comment (jm404) [reply wazuh/cluster/wazuh_cf_master.sh]: Fixed in 48622ea32b939179375d58d3c492001316c3752f

File: splunk/wazuh_cf_splunk.sh (hunk 3 above)
Comment (manuasir) [reply splunk/wazuh_cf_splunk.sh]: My bad.

**Pull request merged**